examples/cms-cosmic/package.json

```
@@ -11,7 +11,7 @@
     "cosmicjs": "^3.2.43",
     "date-fns": "2.14.0",
     "lazysizes": "^5.2.1-rc2",
-    "next": "latest",
+    "next": "^10.0.0",
```

`latest` is Next.js 10
```suggestion
    "next": "latest",
```